polygon-digital-carbon/schema.graphql

```diff
@@ -5,6 +5,9 @@ enum Registry {
   GOLD_STANDARD
   PURO_EARTH
   ICR
+  ECO
+  JPN
```

Comment:
We can remove JPN, JCS will be the registry/standard identifier we use everywhere.